### PR TITLE
Using Habitat Reorg

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kgarmoe@chef.io>

### CODE REVIEW

File: www/source/layouts/sidebar.slim [hunk 1]
```diff
@@ -17,7 +17,7 @@
                           li.main-sidebar--list--item  class=link_classes(current_resource.url, sub_link)
                             = link_to sub_link.title, sub_link.link
           - if sidebar_layout == 'docs'
-            = link_to 'Download the Chef Habitat CLI', '/docs/using-habitat/#install-habitat', class: 'button download'
+            = link_to 'Download the Chef Habitat CLI', '/docs/using-habitat/install-habitat', class: 'button download'
```
Comment (mwrock) [inline www/source/layouts/sidebar.slim]: This link 404s

File: www/source/docs/using-habitat/remote-control.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,102 @@
+---
```
Comment (mwrock) [inline www/source/docs/using-habitat/remote-control.html.md.erb]: I don't think we have a sidebar link for this page?

**Comment by kagarmoe** [comment]
This change should fix the nav @mwrock

**Comment by mwrock** [comment]
still getting 404s. On the download link of the "get started" page. I think you want to do a global search for `docs/install-habitat/"`.

**Pull request merged**